cmake/modules/BuildQATzip.cmake

```diff
@@ -0,0 +1,46 @@
+function(build_qatzip)
+  set(QATzip_REPO https://github.com/intel/qatzip.git)
+  set(QATzip_TAG "v1.1.2")
+
+  include(FindMake)
+  find_make("MAKE_EXECUTABLE" "make_cmd")
+
+  set(QATzip_SOURCE_DIR ${CMAKE_BINARY_DIR}/src/qatzip)
+  set(QATzip_INSTALL_DIR ${QATzip_SOURCE_DIR}/install)
+  set(QATzip_INCLUDE_DIR ${QATzip_INSTALL_DIR}/include)
+  set(QATzip_LIBRARY ${QATzip_INSTALL_DIR}/lib/libqatzip.a)
+
+  # this include directory won't exist until the install step, but the
+  # imported targets need it early for INTERFACE_INCLUDE_DIRECTORIES
+  file(MAKE_DIRECTORY "${QATzip_INCLUDE_DIR}")
+
+  set(configure_cmd env CC=${CMAKE_C_COMPILER} ./configure --prefix=${QATzip_INSTALL_DIR})
```

Comment:
earlier generations of QAT devices (e.g. c62x, dh895xxcc, etc.) are not supported by in-tree qat driver and [qatlib](https://github.com/intel/qatlib), and they only support out-of-tree qat driver. [QATzip](https://github.com/intel/QATzip) installation is different with qatlib and out-of-tree qat driver.